odoo/models.py

```diff
@@ -6301,6 +6321,14 @@ def modified(self, fnames, create=False, before=False):
                     # not stored as this is not really necessary.
                     if field.recursive:
                         recursively_marked = records & self.env.cache.get_records(records, field)
+
+                    if field.type == 'properties' and field.name in fnames:
+                        # TODO: no need to invalidate properties because we
+                        # give the new value in vals (write method)
+                        # Step: change the parent, write on the properties, save
+                        # => traceback because the cache has been invalidated
+                        # Grep in models.py: assert len(values) == len(records)
```

Comment:
If we change the parent on the ticket e.g. and write on the properties fields, both values will be given in `vals` in the write method.

But for some reason, it tried to invalidate the cache of properties (because it depends on the parent which has been changed at the same time I guess ?) and it shouldn't invalidate it (because we give it's value when we wrote)

Reply:
If it can help to test: https://drive.google.com/file/d/1dGfwJqFcqJLwUDMQYXNFCynR_EUpNGIl/view?usp=sharing